src/ImportExport.ts

```diff
@@ -176,6 +178,35 @@ export const importSynergism = async (input: string, reset = false) => {
     }
 }
 
+export const promocodesInfo = async (input: string) => {
+    const textElement = DOMCacheGetOrSet('promocodeinfo');
+    let textMessage = `'${input}': `;
+    let availableUses = 0;
+    switch (input) {
+        case 'daily':
+            textMessage += (player.dailyCodeUsed ? '0' : '1') + ' use left. Next: end of the day.';
```

Comment:
should be "uses" for 0, "use" for 1

Reply:
"uses" for 0, really? English is not my native language, but this seems weird. How is 0 plural?

Reply:
It's weird but correct, https://english.stackexchange.com/q/38293 has a better answer than I could give

Reply:
https://www.quora.com/Why-is-0-plural
Ok it seems to be confirmed. I'm really surprised I've never heard of this before, it feels super werid ^^
Do you want me to change the code again, then? It's already approved, no?

Reply:
Yes please, there are some other comments that I missed below too (you might need to refresh the page)